### ISSUE TITLE
Options for key generator

### ISSUE THREAD

**Issue by infide1** [pfn/passifox#251]
Options for the key generator would be nice (number of characters, special characters etc) Ability to select password preset from keepass's password generator profiles would be nice.

**Comment by hausmaen** [comment]
yes please. I would need an option to reduce password length when generating. some web site reject password due to their limiations. thank you
